ci-operator/step-registry/cucushift/installer/rehearse/azure/aks/hypershift/etcd-disk-encryption/guest/cucushift-installer-rehearse-azure-aks-hypershift-etcd-disk-encryption-guest-workflow.yaml

```diff
@@ -4,6 +4,8 @@ workflow:
     pre:
     - chain: cucushift-installer-rehearse-azure-aks-hypershift-etcd-disk-encryption-provision
     - ref: cucushift-hypershift-extended-enable-guest
+    test:
+    - ref: cucushift-hypershift-extended-debug
```

Comment:
Add a debug step ???

Reply:
This is to test workflow-test as it can't be rehearsed pre-merge. It has no effect on hypershift jobs.

Reply:
You can add anything for debug, but for merge, debug codes should be removed.